superset/connectors/sqla/views.py

```diff
@@ -13,7 +13,7 @@
 from flask_babel import lazy_gettext as _
 from past.builtins import basestring
 
-from superset import appbuilder, db, security, sm, utils
+from superset import appbuilder, db, sm, utils
```

Comment:
Nice! Having both `security` and `sm` was confusing, we should have a single entry point for all security checks and operations.